src/components/hotkeys/main.js

```diff
@@ -0,0 +1,39 @@
+import ZoomInOut from './zoomInOut';
+import store from '@/store';
+import moveShapeByKeypress from '@/components/modeler/moveWithArrowKeys';
```

Comment:
this could be a circular dependency since that function is created in `Modeler.vue` and we are also importing this file into `Modeler.vue`, hence, the circular dependency.
Consider moving this function to this file

Reply:
`moveShapeByKeypress` is created in a separate file that `Modeler.vue` _was_ importing, but since `hotkeys/main.js` is the only file using it now then I agree that `moveWithArrowKeys.js` should be moved elsewhere